### ISSUE TITLE
Is it possible to add the SSD1286 controller to tft_espi

### ISSUE THREAD

**Issue by hannescam** [Bodmer/TFT_eSPI#2672]
I am wondering if it would be possible to add the SSD1286 controller to TFT_eSPI.

**Comment by Bodmer** [comment]
Yes it is possible, but since I do not have one of those displays, so it does not make sense for me to try and add it as any additions to the library must be tested and debugged.
